cleanrl/ppo2_continuous_action.py

```diff
@@ -378,7 +439,13 @@ def forward(self, x):
                                 (1.+args.clip_coef) * advantages,
                                 (1.-args.clip_coef) * advantages).to(device)
 
-        policy_loss = - torch.min(ratio * advantages, clip_adv).mean()
+        # Adding entropy bonus: This is computed using the logprobs and the rsampled actions
+        # so the gradients can also be back propagated through it (?)
+        # Does the entropy function of a distribution backpropagate ? Probably not.
+        # Also, this method does not used the old action sampled from and old policy to get the log probs
+        # Any discussion on what would be the difference ?
```

Comment:
If you are using Pytorch entropy computation, then the sampled actions has nothing to do with the entropy, which itself is calculated solely by the standard deviation of the normal distribution.

Reply:
Thanks. It makes more sense now.
Anyway, let's use the manual computation, as OpenAI Baseline do the same.